Plots/Trajectories/NCL_traj_1.py

```diff
@@ -0,0 +1,86 @@
+"""
```

Comment:
As far as I understand, `Plots/Trajectories/NCL_traj_1.py` is already being implemented within another PR #72 by the branch **traj_1**. The reason for this file `Plots/Trajectories/NCL_traj_1.py` being also changed here in this PR  seems to me that it was first started being implemented by this branch (unique_5) but then left as is.

Correct me if I am wrong and please clarify the reason. Otherwise, could you please remove this file from this PR, then we can handle it in PR #72?